lib/python/docs/notebooks/experiment_tracking_demo.ipynb

```diff
@@ -238,7 +229,9 @@
    "outputs": [],
    "source": [
     "experiment_mlflow = model.create_experiment()\n",
-    "experiment_mlflow.from_mlflow(tracking_uri=\"http://127.0.0.1:5000\", experiment_id=\"\") # <- INSERT MLFLOW EXPERIMENT ID. CAN BE FOUND ON THE UI."
+    "experiment_mlflow.from_mlflow(\n",
+    "    tracking_uri=\"http://127.0.0.1:5000\", experiment_id=\"523964781059248151\"\n",
```

Comment:
Could we redact this id and add a screenshot to show the user where this id coming from?

Reply:
Done